CHANGELOG.md

```diff
@@ -44,68 +19,251 @@ The following are deprecated and will be removed in a future release. For anythi
-- meshUtils.ungeoreference deprecated since version 4.30. Use `convertVertexSpace` instead.
-- Navigation.mouseWheelZoomEnabled deprecated since version 4.32. Use actionMap.mouseWheel instead.
-- NavigationToggle deprecated since 4.32. Use the Navigation Toggle component instead. For information on widget deprecation, read about Esri's move to web components.
-- pointCloudRenderers deprecated since version 4.32. Use unionTypes to import union types, or individual modules to import classes.
-- projection deprecated since version 4.32. Use the projectOperator instead.
-- rasterRenderers deprecated since version 4.32. Use unionTypes to import union types, or individual modules to import classes.
-- RasterStretchRenderer.statistics deprecated since version 4.31. Use customStatistics instead.
-- renderers deprecated since version 4.32. Use unionTypes to import union types, or individual modules to import classes.
-- ScaleBar deprecated since 4.32. Use the Scale Bar component instead. For information on widget deprecation, read about Esri's move to web components.
-- SceneView.highlightOptions deprecated since version 4.32. Use the highlights property instead.
-- Swipe deprecated since 4.32. Use the Swipe component instead. For information on widget deprecation, read about Esri's move to web components.
-- symbols deprecated since version 4.32. Use unionTypes to import union types, or individual modules to import classes.
-- The `"connectivity"` possible value for QueryAssociationsParameters.types is deprecated at 4.29. Please use `"junction-junction-connectivity"` instead.
-- TimeExtent deprecated since version 4.31. Use TimeExtent instead.
-- TimeInterval deprecated since version 4.31. Use TimeInterval instead.
-- Track deprecated since 4.32. Use the Track component instead. For information on widget deprecation, read about Esri's move to web components.
-- UtilityNetworkTrace.gdbVersion deprecated since version 4.31, gdbVersion will be removed and the gdbVersion of the UtilityNetwork will be consumed directly.
-- UtilityNetworkTraceViewModel.gdbVersion deprecated since version 4.31, gdbVersion will be removed and the gdbVersion of the UtilityNetwork will be consumed directly.
-- VersionManagementViewModel.versionIdentifierLookup deprecated since version 4.30. Use VersioningState instead.
-- VersionManagementViewModel.versionInfoLookup deprecated since version 4.30. Use VersioningState instead.
-- VersionManagementViewModel.versionManagementServiceLookup deprecated since version 4.30. Use VersioningState instead.
-- VideoPlayer deprecated since version 4.33. Use the Video Player component instead.
-- View2D.highlightOptions deprecated since version 4.32. Use the highlights property instead.
-- Zoom deprecated since 4.32. Use the Zoom component instead. For information on widget deprecation, read about Esri's move to web components.
+-	Accessor.watch deprecated since version 4.32. Use reactiveUtils.watch instead.
```

Comment:
do you think you could alphabetize this list @andygup ? I think it will make things easier to find.

Reply:
The list seems to be in the same order that's on the SDK site. I'm showing the Accessor.watch line as being at the top of the list. Are you seeing something different? Maybe something is out of whack.